examples/meta/src/binary/multiple_kernel_learning.sg

```diff
@@ -1,49 +1,47 @@
-CSVFile f_feats_train("../../data/classifier_binary_2d_nonlinear_features_train.dat")
-CSVFile f_feats_test("../../data/classifier_binary_2d_nonlinear_features_test.dat")
-CSVFile f_labels_train("../../data/classifier_binary_2d_nonlinear_labels_train.dat")
-CSVFile f_labels_test("../../data/classifier_binary_2d_nonlinear_labels_test.dat")
+File f_feats_train = csv_file("../../data/classifier_binary_2d_nonlinear_features_train.dat")
+File f_feats_test = csv_file("../../data/classifier_binary_2d_nonlinear_features_test.dat")
+File f_labels_train = csv_file("../../data/classifier_binary_2d_nonlinear_labels_train.dat")
+File f_labels_test = csv_file("../../data/classifier_binary_2d_nonlinear_labels_test.dat")
 
 #![create_features]
 Features features_train = features(f_feats_train)
 Features features_test = features(f_feats_test)
-BinaryLabels labels_train(f_labels_train)
-BinaryLabels labels_test(f_labels_test)
+Labels labels_train = labels(f_labels_train)
+Labels labels_test = labels(f_labels_test)
 #![create_features]
 
 #![create_kernel]
-PolyKernel poly_kernel(10,2)
-GaussianKernel gauss_kernel_1(2.0)
-GaussianKernel gauss_kernel_2(3.0)
+Kernel poly_kernel = kernel("PolyKernel", degree=2)
+Kernel gauss_kernel_1 = kernel("GaussianKernel", log_width=0.0)
```

Comment:
due to the changed gaussian kernel parameters, i needed to re-generate the integration testing data.